helm/templates/letsencrypt-issuer.yaml

```
@@ -0,0 +1,16 @@
+apiVersion: cert-manager.io/v1alpha2
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-issuer
+spec:
+  acme:
+    email: jdenen@pillartechnology.com
```

Hmm. I don't _think_ this is the value I want here.

---

```suggestion
    email: johnson.denen@gmail.com
```
Unless there's a specific reason not to update, let's go with this.

Also, should this be configurable? (not a passive suggestion, I legit am unsure)

---

Should be able to template and require that.